examples/simple-viewer/worker.js

```diff
@@ -31,100 +31,56 @@ importScripts("../../lib/mupdf-wasm.js")
 // Import the MuPDF bindings.
 importScripts("../../lib/mupdf.js")
 
-const workerMethods = {}
+const methods = {}
 
 onmessage = async function (event) {
 	let [ func, id, args ] = event.data
 	try {
-		let result = workerMethods[func](...args)
+		let result = methods[func](...args)
 		postMessage([ "RESULT", id, result ])
 	} catch (error) {
-		if (error instanceof mupdf.TryLaterError) {
-			trylaterQueue.push(event)
-		} else {
-			postMessage([ "ERROR", id, { name: error.name, message: error.message, stack: error.stack } ])
-		}
-	}
-}
-
-let trylaterScheduled = false
-let trylaterQueue = []
-mupdf.onFetchCompleted = function (_id) {
-	if (!trylaterScheduled) {
-		trylaterScheduled = true
-		setTimeout(() => {
-			trylaterScheduled = false
-			let currentQueue = trylaterQueue
-			trylaterQueue = []
-			currentQueue.forEach(onmessage)
-		}, 0)
+		postMessage([ "ERROR", id, { name: error.name, message: error.message, stack: error.stack } ])
 	}
 }
 
-let openStream = null
-let openDocument = null
-
-workerMethods.setLogFilters = function (filters) {
-	logFilters = filters
-}
-
-workerMethods.openStreamFromUrl = function (url, contentLength, progressive, prefetch) {
-	openStream = new mupdf.Stream(url, contentLength, Math.max(progressive << 10, 1 << 16), prefetch)
-}
+var openDocument = null
```

Comment:
It would be nice if there was a comment here which said something like "This object will be instance of `Document` - see: https://mupdf.readthedocs.io/en/latest/mutool-run-js-api.html#document ". Just to help developers a little.